pkg/manifests/manifests.go

```diff
@@ -1481,6 +1508,31 @@ func (f *Factory) setupQueryLogFile(p *monv1.Prometheus, queryLogFile string) er
 	return nil
 }
 
+func (f *Factory) setupScrapeProfiles(p *monv1.Prometheus, scrapeProfile ScrapeProfile) error {
+	profiles := make([]string, 0, len(ScrapeProfiles)-1)
+	for _, profile := range ScrapeProfiles {
+		if profile == scrapeProfile {
+			continue
+		}
+		profiles = append(profiles, string(profile))
+	}
+
+	labelSelector := &metav1.LabelSelector{
```

Comment:
Maybe add a comment why this selector operator is chosen?

Reply:
Indeed I added an implementation comment, hopefully, it makes it clear!